src/components/Menu/Menu.stories.args.ts

```diff
@@ -114,4 +114,15 @@ export default {
       },
     },
   },
+  'aria-labelledby': {
```

Comment:
```suggestion
  'ariaLabelledby': {
```
This doesn't work in the story unless it's named the same as the property/attribute.